src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -851,7 +873,7 @@ private SyntaxToken RenameWithinToken(SyntaxToken oldToken, SyntaxToken newToken
                     {
                         if (usingDirective.Alias != null && usingDirective != currentUsing)
                         {
-                            if (usingDirective.Alias.Name.Identifier.ValueText == currentUsing.Alias.Name.Identifier.ValueText)
+                            if (usingDirective.Alias.Name.Identifier.ValueText == currentUsing.Alias!.Name.Identifier.ValueText)
```

Comment:
it's actually not clear to be why `!` is rigth here.  but 🤷

Reply:
Same here - just preserved semantics as I am not sure either...